apps/src/applab/applab.js

```diff
@@ -789,10 +773,11 @@ Applab.init = function (config) {
     showDebugButtons: showDebugButtons,
     showDebugConsole: showDebugConsole,
     showDebugWatch: false,
-    localeDirection: studioApp.localeDirection()
+    localeDirection: studioApp.localeDirection(),
+    isDroplet: true
```

Comment:
This value gets used in exactly one place, to set a property called `editCode` on `ConnectedCodeWorkspace`.  Not really an issue, just wondering if there's a nice way around knowing/caring about this globally.